src/reference-designs/tko-data-protection.md

```diff
@@ -0,0 +1,38 @@
+
+Data Protection for Tanzu Kubernetes Grid workloads is provided by the Tanzu Mission Control data protection feature which is built upon an open-source foundation using the [Velero project](https://velero.io/), the most popular open-source project for Kubernetes data protection.
+
+Using Tanzu Mission Control Data protection, you can back up and restore both stateless and stateful applications. Backups can be stored on AWS S3 or on any compatible S3 storage, giving you the flexibility to bring your own storage.
+
+Using the Tanzu Mission Control UI, CLI, or API, you can create backups and restores for all of your clusters, regardless of where they are located, from a central management platform. Backups are configured at the cluster level and can include:
+
+- an individual namespace
+- a group of namespaces or all namespaces
+- all resources in a cluster
+- specific resources in a cluster identified by a given label
+
+You can selectively restore the backups you have created by specifying any of the following:
+
+- the entire backup
+- selected namespaces from the backup
+- specific resources from the backup identified by a given label
+
+You can also configure backup scheduling. You can specify how often to back up the Kubernetes data and how long to retain your backups. When the retention period for a backup expires, Tanzu Mission Control data protection automatically removes the expired backup from storage, lowering your overall storage usage.
+
+When it's time to destroy a cluster, Tanzu Mission Control gives you the choice of deleting your backup storage for that cluster or saving it for future data recoveries to a new cluster.
+
+When you enable data protection for a cluster, Tanzu Mission Control installs Velero with restic (an open-source backup tool), configured to use the opt-in approach. With this approach, you must annotate the pods whose volumes you want to include in backups. <_Is this info correct for the current version of Valero? Is it still necessary to annotate pods?_>
+
+- If a given pod is annotated to be backed up, persistent volume snapshots are included in the backup.
+- If a pod is not annotated to be backed up, restic does not include the persistent volume snapshots in the backup. However, Velero still creates snapshots if the specified target location for the backup is in the same cloud provider account as the cluster.
+
+Tanzu Mission Control manages the entire Valero lifecycle automatically, eliminating the need for you to manually update or upgrade it when newer software releases are available.
+
+If Tanzu Mission Control data protection does not handle all of your backup requirements, there is also a standalone version of Velero that you can install on your Tanzu Kubernetes clusters. Using the Velero standalone is advisable in the following scenarios:
+
+- In an internet-restricted or airgap environment, where the Tanzu Kubernetes clusters are not connected to the internet.
```

Comment:
"air-gapped" please

Reply:
Craig, airgapped, without the hyphen is good.